rc.js

```diff
@@ -51,6 +53,8 @@ module.exports = function (pkg) {
 
   rc.abi = napi.isNapiRuntime(rc.runtime) ? rc.target : getAbi(rc.target, rc.runtime)
 
+  rc.libc = rc.libc === detectLibc.GLIBC ? '' : rc.libc
```

Comment:
Could be a breaking change if someone is setting the `LIBC` env var?

Reply:
Thanks! I figured there was another place deeper in the code reading env.LIBC. I fixed the issue and pushed update to my branch. I'm going to write a longer reply below how I verified my fix should be doing what I say.